homeassistant/components/device_tracker/strings.json

```diff
@@ -53,7 +53,7 @@
         },
         "dev_id": {
           "name": "Device ID",
-          "description": "ID of the device (find the ID in `known_devices.yaml`)."
+          "description": "ID of the device."
```

Comment:
The legacy see service is for legacy platforms where known_devices.yaml is still used.

Reply:
But the service is not specifically tied to legacy use. I can just as well use it without "known_devices.yaml" to "see" regular non-legacy entities. The current service field description makes it sound like you have to have that file.
The documentation is also not referencing the "known_devices.yaml" anymore for the service (the page itself of course mentions that this file existed a long time ago).

https://www.home-assistant.io/integrations/device_tracker/#device_trackersee-service

Reply:
You can't use it on non legacy entities that are part of the entity registry. If a device is not already part of known_devices.yaml it will be added when it's seen.